src/renderer/components/FtKeyboardShortcutPrompt/FtKeyboardShortcutPrompt.vue

```diff
@@ -0,0 +1,184 @@
+
+const localizedShortcutNameDictionary = computed(() => {
+  return new Map([
+    ['SHOW_SHORTCUTS', t('KeyboardShortcutPrompt.Show Keyboard Shortcuts')],
+    ['HISTORY_BACKWARD', t('KeyboardShortcutPrompt.History Backward')],
+    ['HISTORY_FORWARD', t('KeyboardShortcutPrompt.History Forward')],
+    ['FULLSCREEN', t('KeyboardShortcutPrompt.Fullscreen')],
+    ['NAVIGATE_TO_SETTINGS', t('KeyboardShortcutPrompt.Navigate to Settings')],
+    (
+      isMac
+        ? ['NAVIGATE_TO_HISTORY_MAC', t('KeyboardShortcutPrompt.Navigate to History')]
+        : ['NAVIGATE_TO_HISTORY', t('KeyboardShortcutPrompt.Navigate to History')]
+    ),
+    ['NEW_WINDOW', t('KeyboardShortcutPrompt.New Window')],
+    ['MINIMIZE_WINDOW', t('KeyboardShortcutPrompt.Minimize Window')],
+    ['CLOSE_WINDOW', t('KeyboardShortcutPrompt.Close Window')],
+    ['RESTART_WINDOW', t('KeyboardShortcutPrompt.Restart Window')],
+    ['FORCE_RESTART_WINDOW', t('KeyboardShortcutPrompt.Force Restart Window')],
+    ['TOGGLE_DEVTOOLS', t('KeyboardShortcutPrompt.Toggle Developer Tools')],
+    ['RESET_ZOOM', t('KeyboardShortcutPrompt.Reset Zoom')],
+    ['ZOOM_IN', t('KeyboardShortcutPrompt.Zoom In')],
+    ['ZOOM_OUT', t('KeyboardShortcutPrompt.Zoom Out')],
+    ['FOCUS_SEARCH', t('KeyboardShortcutPrompt.Focus Search')],
+    ['SEARCH_IN_NEW_WINDOW', t('KeyboardShortcutPrompt.Search in New Window')],
+
+    ['REFRESH', t('KeyboardShortcutPrompt.Refresh')],
+    ['FOCUS_SECONDARY_SEARCH', t('KeyboardShortcutPrompt.Focus Secondary Search')],
+
+    ['CAPTIONS', t('KeyboardShortcutPrompt.Captions')],
+    ['THEATRE_MODE', t('KeyboardShortcutPrompt.Theatre Mode')],
+    ['FULLSCREEN', t('KeyboardShortcutPrompt.Fullscreen')],
+    ['FULLWINDOW', t('KeyboardShortcutPrompt.Full Window')],
+    ['PICTURE_IN_PICTURE', t('KeyboardShortcutPrompt.Picture in Picture')],
+    ['MUTE', t('KeyboardShortcutPrompt.Mute')],
+    ['VOLUME_UP', t('KeyboardShortcutPrompt.Volume Up')],
+    ['VOLUME_DOWN', t('KeyboardShortcutPrompt.Volume Down')],
+    ['TAKE_SCREENSHOT', t('KeyboardShortcutPrompt.Take Screenshot')],
+    ['STATS', t('KeyboardShortcutPrompt.Stats')],
+
+    ['PLAY', t('KeyboardShortcutPrompt.Play')],
+    ['LARGE_REWIND', t('KeyboardShortcutPrompt.Large Rewind')],
+    ['LARGE_FAST_FORWARD', t('KeyboardShortcutPrompt.Large Fast Forward')],
+    ['SMALL_REWIND', t('KeyboardShortcutPrompt.Small Rewind')],
+    ['SMALL_FAST_FORWARD', t('KeyboardShortcutPrompt.Small Fast Forward')],
+    ['DECREASE_VIDEO_SPEED', t('KeyboardShortcutPrompt.Decrease Video Speed')],
+    ['INCREASE_VIDEO_SPEED', t('KeyboardShortcutPrompt.Increase Video Speed')],
+    ['SKIP_N_TENTHS', t('KeyboardShortcutPrompt.Skip by Tenths')],
+    ['LAST_CHAPTER', t('KeyboardShortcutPrompt.Last Chapter')],
+    ['NEXT_CHAPTER', t('KeyboardShortcutPrompt.Next Chapter')],
+    ['LAST_FRAME', t('KeyboardShortcutPrompt.Last Frame')],
+    ['NEXT_FRAME', t('KeyboardShortcutPrompt.Next Frame')],
```

Comment:
Note to future self: Look into if there is a way to do this, without having to having to use strings as the keys, to reduce the bundle size (also forces terser to bail-out of most optimisations on the `KeyboardShortcuts` constant e.g. mangling the names and hoisting them out of the object).

Reply:
As a note, this representation might change a bit if we implement #251, but I imagine we'll keep the keys around. I don't have any intention on implementing it currently, but my thought would be we have each action has a unique name, each action+name is mapped to a unique shortcut combo, and we replace a lot of our keydown logic with a combo-reading util function that accepts a list of accepted names as arguments. The localization key in that case could either come from putting the name through a de-snake-case-ifier or as another property on the constant object.